themes/cardinal_service/pdb_components/Components/Filters.jsx

```diff
@@ -82,6 +82,7 @@ export class Filters extends Component {
 
   constructor(props) {
     super(props);
+    this.props.apiParams = props.apiParams || ""
```

Comment:
@pookmish I get this error: "Uncaught TypeError: Cannot assign to read only property 'apiParams' of object '#<Object>'" in the browser console.

When I change this line to `this.apiParams = props.apiParams || "";` it works for me.

Reply:
Actually, you can probably remove that line entirely since it's checked down on line 117.

Reply:
That works too.